devsetup/scripts/standalone.sh

```diff
@@ -95,6 +97,9 @@ scp $SSH_OPT standalone/deployed_network.yaml root@$IP:/tmp/deployed_network.yam
 scp $SSH_OPT standalone/network.sh root@$IP:/tmp/network.sh
 scp $SSH_OPT standalone/ceph.sh root@$IP:/tmp/ceph.sh
 scp $SSH_OPT standalone/openstack.sh root@$IP:/tmp/openstack.sh
+if [[ -f /home/zuul/containers-prepare-parameters.yaml ]]; then
```

Comment:
I think we should replace here `/home/zuul` by `$HOME`, as people are using the standalone target to test adoption in scenarios different than CI

Reply:
ack will do thanks